### DISCUSSION TITLE
LayerMask 관련 문제

### DISCUSSION THREAD

**Discussion by natural-jogi** [naverz/zepeto-studio-kor#123] [Issue Report (Legacy)]
GameObject 클릭 시 나오는 Inspector 창에서 LayerMask 값을 넣어줘도 테스트로 시작(▶)을 누르면 Nothing으로 값이 없어집니다.

~~덧붙여, number형 값을 LayerMask형으로 형변환을 어떻게 해야할지도 문의드립니다. (C#에서는 암시적 형변환이 됨)~~
-> LayerMask.op_Implicit 찾았습니다

lm.ts
```
import { LayerMask } from 'UnityEngine'
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class lm extends ZepetoScriptBehaviour {
    public test: LayerMask;
}
```

![image](https://user-images.githubusercontent.com/67745779/147179156-1f9a0342-c7ec-4e25-a100-b2857ee716ac.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 관련 내용 확인해 보겠습니다.
